src/EnergyPlus/CurveManager.hh

```diff
@@ -207,6 +210,17 @@ namespace Curve {
                                        const Real64 Var4, // 4th independent variable
                                        const Real64 Var5, // 5th independent variable
                                        const Real64 Var6);
+
+        void markUsed(EnergyPlusData &state)
```

Comment:
I don't like this. Leave it for now but I'm going to look at how `InputProcessor` is doing this.